draft-ietf-rats-corim.md

```diff
@@ -991,7 +991,19 @@ computing base.
 
 ###### Raw Values Types {#sec-comid-raw-value-types}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/9
+Raw value measurements are typically vendor defined values that are checked by Verifiers
+for consistency only, since the security relevance is opaque to Verifiers.
+
+There are two parts to a `raw-value-group`, a measurement and an optional mask.
```

Comment:
> optional mask

Just looking at this PR and not the whole document: is it clear what mask means here?

Reply:
issue #9 will be worked further